DAOS-4475 test: Updated pool size and container UUID (#2638)

shimizukko opened the pull request:
Use correct container UUID at open.

Updated copyright year.

Test-tag: pr,-hw closehandle
Signed-off-by: Makito Kano <makito.kano@intel.com>

daosbuild1 commented:
Test stage Functional_Hardware_Small completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-2665/2/execution/node/662/log

daosbuild1 commented:
Test stage Functional_Hardware_Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-2665/2/execution/node/780/log

shimizukko commented:
Failed Test:
Functional_Hardware_Small / 1-./io/hdf5.py:Hdf5.test_hdf5;client_processes-container-hosts-pool-servers-test_repo-cea2 – Hdf5 -> DAOS-4796. Fixed in master.
Functional / 08-./daos_test/daos_core_test.py:DaosCoreTest.test_subtest;test_i-num_clients-num_replicas-hosts-server_config-5ae4 – DaosCoreTest -> DAOS-4538. Fixed in master.
